src/Console/Command/Git/InitCommand.php

```diff
@@ -123,11 +123,16 @@ public function execute(InputInterface $input, OutputInterface $output)
     protected function parseHookBody($hook, SplFileInfo $templateFile)
     {
         $content = $this->filesystem->readFromFileInfo($templateFile);
+
         $replacements = [
             '${HOOK_EXEC_PATH}' => $this->paths()->getGitHookExecutionPath(),
             '$(HOOK_COMMAND)' => $this->generateHookCommand('git:' . $hook),
         ];
 
+        foreach ($this->grumPHP->getGitHookVariables() as $key => $value) {
+            $replacements[sprintf('$(%s)', $key)] = $value;
```

Comment:
Values should be escaped since paths can contain some special characters.
I guess you could use `\GrumPHP\Process\ProcessUtils::escapeArgument()` for this.